### PR TITLE
Work for release 0.23.x

### PR DESCRIPTION
**Pull request by pierre**

### CODE REVIEW

File: .github/workflows/docker_ci.yml [hunk 1]
```diff
@@ -7,8 +7,8 @@ on:
 
 env:
   # Doesn't matter too much, we just want to verify the build passes
-  CURRENT_LTS: '0.22'
-  KB_VERSION: '0.22.26'
+  CURRENT_LTS: '0.23'
```
Comment (reshmabidikar) [inline .github/workflows/docker_ci.yml:10]: Is this correct or do we need `0.24` here?

**Review by reshmabidikar — APPROVED** [review]

File: .github/workflows/docker_ci.yml [hunk 2]
```diff
@@ -7,8 +7,8 @@ on:
...
+  KB_VERSION: '0.23.0-ea72fab-SNAPSHOT'
```
Comment (reshmabidikar) [inline .github/workflows/docker_ci.yml:11]: Same comment here, is `0.23.0-ea72fab-SNAPSHOT` correct?

File: .github/workflows/docker_ci.yml (hunk 1 above)
Comment (pierre) [reply .github/workflows/docker_ci.yml:10]: Good catch, we need 0.24 here (and below).

**Comment by pierre** [comment]
Merging to update the Ansible playbooks in `master`. Docker issues still to be figured out.

**Pull request merged**